documentation/asciidoc/topics/attributes/product-attributes.adoc

```diff
@@ -18,6 +18,9 @@
 
 :github_root: https://github.com/infinispan/infinispan-operator/
 :github_raw: https://raw.githubusercontent.com/infinispan/infinispan-operator/
+:github_tag: 0.3.0
```

Comment:
In master, these tags should point to master. When we do releases, e.g. 0.3.1, we can fix them to 0.3.1.